index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile First Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header class="main-header">
+
+            <nav class="header">
+
+                <div class="name"><a href="#" style="color: white; text-decoration: none;">Paul Orife</a></div>
+                <div class="menu"><i class="fas fa-bars"></i></div>
```

Comment:
- [ ] As requested by the previous reviewers, Your menu icon should be found at a minimum width of `375px`. Please note that your mobile version should be responsive until the desktop screen size.❗ But as you can see from the screenshot below your menu icon cant be found.

  ![image](https://user-images.githubusercontent.com/92303891/204999226-bbe6584a-b06c-48ac-a2e1-56aae891beab.png)